CHANGELOG.md

```diff
@@ -15,9 +15,17 @@ Types of changes
 ## [Unreleased]
 
 ### Added
+- Added region folding [Pull #73](https://github.com/isg-stuttgart/vscode-isg-cnc/pull/73)
+- "Format Code" command now indents multilines
+- Autogenerate github pages at push to main branch
+- Filter option to either show grouped or line-by-line sorting of matches in sidebar [Pull #79](https://github.com/isg-stuttgart/vscode-isg-cnc/pull/79)
 
 ### Fixed
-
+- Sidebar cuts off all matches over 500 to prevent performance issues [Pull #74](https://github.com/isg-stuttgart/vscode-isg-cnc/pull/74)
+- Reinclude fixed docu feature [Issue #58](https://github.com/isg-stuttgart/vscode-isg-cnc/issues/58)
+- Add toLowerCase for nc file detection
```

Comment:
Can you please add the diff between the versions at the end of this file.

Reply:
Done